File: app/src/main/java/com/example/mohassu/MyPageFragment/MyPageSettingNotificationFragment.java
//package com.example.mohassu.MyPageFragment;
//
//import android.os.Bundle;
//import android.view.LayoutInflater;
//import android.view.View;
//import android.view.ViewGroup;
//import android.widget.Button;
//
//import androidx.fragment.app.Fragment;
//import androidx.navigation.NavController;
//import androidx.navigation.Navigation;
//
//import com.example.mohassu.R;
//
//public class MyPageSettingNotificationFragment extends Fragment {
//    @Override
//    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
//        return inflater.inflate(R.layout.fragment_mypage_setting_notification, container, false);
//    }
//
//    @Override
//    public void onViewCreated(View view, Bundle savedInstanceState) {
//        super.onViewCreated(view, savedInstanceState);
//
//        // NavController 초기화
//        NavController navController = Navigation.findNavController(view);
//
//        // 뒤로가기 버튼에 클릭 리스너 추가
//        view.findViewById(R.id.btnBack).setOnClickListener(v -> {
//            navController.navigateUp();
//        });
//    }
//}

File: app/src/main/java/com/example/mohassu/MyPageFragment/MyPageSettingNotificationFragment.kt
package com.example.mohassu.MyPageFragment

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import androidx.navigation.NavController
import androidx.navigation.fragment.findNavController
import com.example.mohassu.databinding.FragmentMypageProfileEdit2Binding
import com.example.mohassu.databinding.FragmentMypageSettingNotificationBinding

class MyPageSettingNotificationFragment : Fragment() {
    private var _binding: FragmentMypageSettingNotificationBinding? = null
    private val binding get() = _binding!!

    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?
    ): View {
        _binding = FragmentMypageSettingNotificationBinding.inflate(inflater, container, false)
        return binding.root
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)

        val navController = findNavController()
        binding.btnBack.setOnClickListener { navController.navigateUp() }
    }

    override fun onDestroyView() {
        super.onDestroyView()
        _binding = null
    }
}
